controllers/podcast.js

```diff
@@ -1,86 +1,86 @@
 const {
-    updatePodcastDetails,
-    getPodcastById,
-    savePodcastToDb,
-    deletePodcastFromDb,
-    getPodcastSearchResults,
-    getBestPodcasts
+  updatePodcastDetails,
+  getPodcastById,
+  savePodcastToDb,
+  deletePodcastFromDb,
+  getPodcastSearchResults,
+  getBestPodcasts
 } = require('../services/podcast')
 
 const getBestPodcast = async (req, res, next) => {
-    const numberOfPodcasts = parseInt(req.params.number)
-    try {
-        const podcastsInfo = await getBestPodcasts(numberOfPodcasts)
-        if (!podcastsInfo) return res.status(404).send('No podcasts to present')
-        return res.status(200).send(podcastsInfo)
-    } catch
-        (err) {
-        return next(err)
-    }
+  const numberOfPodcasts = parseInt(req.params.number)
+  try {
+    const podcastsInfo = await getBestPodcasts(numberOfPodcasts)
+    if (!podcastsInfo) return res.status(404).send('No podcasts to present')
+    return res.status(200).send(podcastsInfo)
+  } catch
+    (err) {
```

Comment:
indentation...why is the (err) in a new line?

Reply:
eslint at it again...